Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import './App.css';
+import HeaderContainer from './components/HeaderComponents/HeaderContainer.js';
+import CardContainer from './components/CardComponents/CardContainer';
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Social Card!</h3>
+     <div className="appContainer">
```

Comment:
It is more common to use dashes instead of camelCase for *className*. ex. *app-container*.